frontend/src/lib/components/CompareFilter/compareFilterLogic.ts

```diff
@@ -0,0 +1,41 @@
+import { dayjs } from 'lib/dayjs'
+
+import { dateStringToDayJs } from '~/lib/utils'
+import type { DateRange } from '~/queries/schema'
+import type { IntervalType } from '~/types'
+
+const DEFAULT_RELATIVE_START_DATE = '1d'
+
+export const getDefaultComparisonPeriodRelativeStartDate = (
```

Comment:
This method is based on [getComparePeriodDates](https://github.com/PostHog/posthog/blob/c72f8dc76b62b969c51df6a4256626947a231df3/posthog/utils.py#L547). It's simpler and seems more consistent between "last 7 days" and "last 6 days"